meta-ostro-xt/recipes-opencl/ocl-icd/ocl-icd/configurable-doc-generation.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/configure.ac b/configure.ac
```

Comment:
that's the reason for clean build inside container: catch inconsistencies in data and patches :)